### PR TITLE
Overhaul of Octant storybook implementation

### PR DESCRIPTION
**Pull request by mklanjsek**
The intention of this work was to provide better performance of our Storybook portal by using the newly released Storybook inline mode for angular components. However, it turned out that much more serious surgery was required to make everything working at that level, resulting in pretty much overhaul of internal Storybook organization.

- Bumped Storybook to latest 6.3 build
- Switched to inline story mode, so stories are now not displayed in iFrame, resulting in much better update performance. This required a few small fixes in our code base
- Modified each story to stop using the old `storiesOff` model and use the hoisted annotations
- Upgraded the component properties editing by replacing soon to be removed `knobs` addon (has issues with inline stories) with the new generation `controls` addon that supports editing in both Docs and Canvas mode
- Added the Accessibility pane so we can start keeping an eye on component accessibility
- Lots of small tweaks and improvements

### CODE REVIEW

**Comment by GuessWhoSamFoo** [comment]
The deploy preview is failing because this needs a rebase to include the nelify.toml move from eb94d630b1b89b1e6e760f8005427b3d651ccbdb

**Review by GuessWhoSamFoo — CHANGES_REQUESTED** [review]

File: web/src/stories/graph.stories.ts [hunk 1]
```diff
@@ -12,6 +10,11 @@ import {
   Service,
   ServiceAccount,
 } from '../app/modules/shared/components/presentation/cytoscape2/shape';
+import { Meta, Story } from '@storybook/angular/types-6-0';
+
+export default {
```
[the remark this answers is not in the document]
Comment (mklanjsek) [reply web/src/stories/graph.stories.ts]: Good point, `Other/Resources` was double entry, it's fixed now.

File: web/src/stories/timeline.stories.mdx [hunk 2]
```diff
@@ -41,46 +41,55 @@ export const buttonGroupView = {
...
+    type: 'timeline',
+  },
+  config: {
+    steps: [
+      {
+        state: "current",
+        header: "current header",
+        title: 'Step 1',
+        description: 'this is a current step',
+        buttonGroup: buttonGroupView,
```
[the remark this answers is not in the document]
Comment (mklanjsek) [reply web/src/stories/timeline.stories.mdx:59]: Updated button definition

File: web/src/stories/accordion.stories.mdx [hunk 3]
```diff
@@ -14,68 +14,78 @@ export const accordionDocs= { source: { code: `	accordion := component.NewAccord
...
-                },
-              },
-            ],
-          },
-        }),
-      },
-      component: AccordionComponent,
-    }}
+  <Story name="Accordion component"
+         component={AccordionComponent}
```
[the remark this answers is not in the document]
Comment (mklanjsek) [reply web/src/stories/accordion.stories.mdx]: You are right, I missed that. All Component entries are now removed from MDX stories.

**Comment by mklanjsek** [comment]
@GuessWhoSamFoo PTAL

**Review by GuessWhoSamFoo — APPROVED** [review]
We are still seeing warnings for `storyFn` at 6.3: https://github.com/storybookjs/storybook/issues/13074

Not a blocker, but something to watch when storybook 7 comes out.

**Comment by mklanjsek** [comment]
Sounds good, I am not seeing these but will keep an eye on them.

**Pull request merged**